src/test/app/MultiSign_test.cpp

```diff
@@ -190,7 +190,12 @@ class MultiSign_test : public beast::unit_test::suite
     test_enablement()
     {
         using namespace jtx;
-        Env env(*this, FeatureBitset{});
+        Env env(*this, envconfig([](std::unique_ptr<Config> cfg)
```

Comment:
for your consideration: here's a commit that uses an `envconfig` helper for flipping this setting:  https://github.com/mellery451/rippled/commit/d440918cb9ab71ba362377544a7a4ae59bfd806f